src/display/annotation_layer.js

```diff
@@ -620,7 +621,8 @@ class ChoiceWidgetAnnotationElement extends WidgetAnnotationElement {
 
     const selectElement = document.createElement("select");
     selectElement.disabled = this.data.readOnly;
-
+    selectElement.setAttribute("name", this.data.fieldName);
+    
```

Comment:
Linting is complaining about whitespace on this line.

Reply:
Fixed